Update Formik to v2.4.2, Add React-Toastify, Fix Video Upload Form, Update Video List Page, and Server-side Changes

hanifsheikh opened the pull request:
1. Updated Formik to version 2.4.2
2. Added React-Toastify package
3. Removed unnecessary imports
4. Video Upload form updated and fixes console warnings. #45 
5. Video List Page Items Key prop changed id to _id. 
6. In server-side updated .env.template (SERVER_URL=http://localhost:4000) to fix Thumbnail not found issue. updated video schema language field data from full-form string to short-form enums like 'Bangla' as 'bn' to fix the mongoDB language override error that stops uploading/processing the video. #43 
7. Get data order by 'recordingDate' DESC  from videos collections from server-side #44

foyzulkarim commented:
Thanks for your pull request. Here are some feedbacks:

- Make the title reflect (or summarize) of your changes. 
- create issue and associate the issue with this PR
- try  not to change server side code because i have some different plans for them (for now it is fine)

hanifsheikh commented:
> Thanks for your pull request. Here are some feedbacks:
> 
> * Make the title reflect (or summarize) of your changes.
> * create issue and associate the issue with this PR
> * try  not to change server side code because i have some different plans for them (for now it is fine)

I have added the description and will not change any server-side code.

hanifsheikh commented:
1. Updated Formik to version 2.4.2
2. Added React-Toastify package
3. Removed unnecessary imports
4. Video Upload form updated and fixes console warnings. #45 
5. Video List Page Items Key prop changed id to _id. 
6. In server-side updated .env.template (SERVER_URL=http://localhost:4000) to fix Thumbnail not found issue. updated video schema language field data from full-form string to short-form enums like 'Bangla' as 'bn' to fix the mongoDB language override error that stops uploading/processing the video. #43 
7. Get data order by 'recordingDate' DESC  from videos collections from server-side #44
